pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -30,11 +30,17 @@ let
     sha256 = "sha256-BYL7wxsYRI45l8C3VwxYIIocn5TzJnBtU0UZ9pHwwZw=";
   };
 
-  eigen = fetchFromGitLab {
-    owner = "libeigen";
-    repo = "eigen";
-    rev = "d10b27fe37736d2944630ecd7557cefa95cf87c9";
-    sha256 = "sha256-Lmco0s9gIm9sIw7lCr5Iewye3RmrHEE4HLfyzRkQCm0=";
+  eigen' = applyPatches {
+    name = "patch-build-error.patch";
+    src = eigen.src;
+    patches = eigen.patches or [] ++ [
+      # Patch for Eigen seems to be necessary to avoid build errors as referenced here:
+      # https://aur.archlinux.org/pkgbase/python-onnxruntime?O=30#comment-856116
```

Comment:
404

Reply:
I think because python-onnxruntime was moved out of AUR into extra. Unfortunately the comments are now gone it seems :'(

Internet archive to the rescue: https://web.archive.org/web/20240121014111/https://aur.archlinux.org/pkgbase/python-onnxruntime?O=40#comment-856116